core/sqf/sqenvcom.sh

```diff
@@ -795,13 +799,9 @@ fi
 
 # Non-standard or newer version tools
 export BISON="${TOOLSDIR}/bison_3_linux/bin/bison"     # Need 1.3 version or later
-
-
 export LLVM="${TOOLSDIR}/dest-llvm-3.2"
 export UDIS86="${TOOLSDIR}/udis86-1.7.2"
 export ICU="${TOOLSDIR}/icu4.4"
-export QT_TOOLKIT="${TOOLSDIR}/Qt-4.8.5-64"
-
 
```

Comment:
In my next checkin, I am planning to remove the checks for QT_TOOLKIT from the $MY_SQROOT/../bldenvchk.sh script. I'll also remove the check for the Hadoop lib and include directories.

If you want you can take out the QT_TOOLKIT from sqenvcom.sh and I can put it back in once I updated bldenvchk.sh.